### PR TITLE
Kubernetes: update domain list for firewall config

### PR DESCRIPTION
**Pull request by glaubinix**
I reordered the domain to match the list on the Replicated docs based off https://docs.replicated.com/enterprise/installing-general-requirements#firewall-openings-for-online-installations

### CODE REVIEW

File: docs/self-hosted/kubernetes-embedded.md [hunk 1]
```diff
@@ -24,14 +24,19 @@ in an existing Kubernetes cluster using Helm follow [this guide](./kubernetes-he
     * Ports 80, 443, and 8800 must be accessible, for a full list of ports
     * Must be reachable at the chosen domain names from your local machine
 1. If your firewall restricts external connections the following domains must be accessible from the server:
-  * hub.docker.com
-  * proxy.replicated.com
+  * index.docker.io
```
Comment (IgorBenko) [inline docs/self-hosted/kubernetes-embedded.md:27]: Is this still needed with the *.docker.io below?

File: docs/self-hosted/kubernetes-helm.md [hunk 2]
```diff
@@ -13,12 +13,13 @@ to instead install Private Packagist Self-Hosted without an existing Kubernetes
 1. An SSL certificate valid for both chosen domains
 1. An SMTP server or a GMail account for Private Packagist Self-Hosted to send email
 1. If your firewall restricts external connections then the following domains must be accessible from the server:
-  * hub.docker.com
+  * index.docker.io
```
Comment (IgorBenko) [inline docs/self-hosted/kubernetes-helm.md:16]: Same here

**Review by IgorBenko — CHANGES_REQUESTED** [review]

File: docs/self-hosted/kubernetes-embedded.md (hunk 1 above)
Comment (glaubinix) [reply docs/self-hosted/kubernetes-embedded.md:27]: Replicated lists them separate. My understanding is that it isn't but was just trying to follow this as closely as possible
Screenshot 2024-11-22 at 17 05 53

Comment (IgorBenko) [reply docs/self-hosted/kubernetes-embedded.md:27]: Yeah, just noticed that too in the docs. Then let's keep it like that

**Review by IgorBenko — APPROVED** [review]

**Pull request merged**